k8s/trillian/manifest/etcd.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: $NAME-etcd-service
+    labels: &EtcdLabels
+      app.kubernetes.io/name: "$NAME"
+      app.kubernetes.io/component: etcd
+  spec:
+    type: NodePort
+    selector: *EtcdLabels
+    ports:
+      - protocol: TCP
+        port: 2379
+        nodePort: 32379
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: $NAME-etcd-operator
```

Comment:
If we deleted the application, should the etcd cluster created by this operator be also deleted?

Reply:
Yes, it should. However, I've just noticed that [deleting the etcd-operator doesn't delete the cluster](https://github.com/coreos/etcd-operator/blob/master/doc/user/install_guide.md#uninstall-etcd-operator). Is there something I need to add to the manifests to achieve this?

Reply:
I've just tested and the etcd cluster is successfully deleted along with the operator.